scheduler/scheduler.go

```diff
@@ -147,6 +147,20 @@ func SchedHandler(scheduler string) interface{} {
 	return reflect.New(reflect.TypeOf(schedType)).Interface()
 }
 
+// RegisteredActions returns a list of currently registered actions
+func RegisteredSchedulers() []string {
+	shLock.Lock()
+	defer shLock.Unlock()
+
+	schedulerList := make([]string, len(schedulerHandler))
+	i := 0
+	for registeredScheduler := range schedulerHandler {
```

Comment:
Why not use:

`for i, registeredScheduler := range schedulerHandler {`

Reply:
because `schedulerHandler` is a `map[string]IScheduler`